CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to this repository
+
+# Audience
+This set of contributing guidelines is for both external contributors, as well as Verne Global employees. There are number of references to using JIRA issue number in this document. If you are an external contributor and do not have access to the Verne Global JIRA instance, please ignore these references.
+
+# Table of Contents
+1. [Commits and pull requests](#commits-release-notes-pull-requests)
+    1. [Fixing a bug](#fixing-a-bug)
+    2. [Commits](#commits)
+    3. [Pull requests](#pull-requests)
+        1. [Reviewing pull requests](#reviewing-pull-requests)
+        2. [Merging pull requests](#merging-pull-requests)
+
+## Commits and pull requests
+### Fixing a bug
+
+1. Clone the repo, and commit changes to a new branch, named using the convention <JIRA-ISSUE-NUMBER>-description-of-change. e.g. ```HPCDIR-9999-add-contributing```
+2. Make pull requests against the master branch in the repo.
+3. Reference the JIRA issue in both the commit message and the pull request message.
+
+### Commits
+
+Please avoid the following in a commit:
+
+* Mixing whitespace changes with functional code changes.
```

Comment:
Agreed, but how do you propose we deal with any whitespace or general "lint" fixes? Do they get their own pull/merge request? Does that warrant going through code review?

Reply:
No, they can be in the same PR/MR, just a separate commit (i.e. multiple commits in a single MR?PR), to ensure each change is atomic enough to be reverted if needs be.

Reply:
ah right, gotcha